python/src/aiconfig/editor/client/src/utils/promptUtils.ts

```diff
@@ -84,6 +86,12 @@ export const PROMPT_SCHEMAS: Record<string, PromptSchema> = {
 
   // Local HuggingFace Parsers
   HuggingFaceText2ImageDiffusor: HuggingFaceText2ImageDiffusorPromptSchema,
+  TextGeneration: HuggingFaceTextGenerationTransformerPromptSchema,
+  Text2Image: HuggingFaceText2ImageDiffusorPromptSchema,
+  AutomaticSpeechRecognition: HuggingFaceAutomaticSpeechRecognitionPromptSchema,
```

Comment:
Nice yea, we'll just hardcode these names for now

Reply:
We'll want to change them to whatever the hf package is using when registering them, but not sure what that final copy will be just yet

cc @saqadri for visibility